packages/contracts/contracts/AssetWrappers/Interfaces/IWAsset.sol

```diff
@@ -18,11 +18,11 @@ interface IWAsset {
 
     function claimRewardFor(address _for) external;
 
-    function getPendingRewards(address _for) external view returns (address[] memory tokens, uint[] memory amounts);
+    function getPendingRewards(address _for) external view returns (address[] memory, uint[] memory);
```

Comment:
This is fine to keep, we just don't want it in implementations. It would be more confusing if we removed arg names from interfaces